src/renderer/css/index.scss

```diff
@@ -0,0 +1,219 @@
+@import url("fonts.css");
+@import '~@mattermost/compass-icons/css/compass-icons.css';
+
+#app {
+  height: 100%;
+}
+
+#main {
+
+  //height: 100%;
+  //font-size: 14px;
+  //min-height: 100%;
+
+  position: absolute;
+  width: 100%;
+
+  .hovering-enter {
```

Comment:
I would avoid specifying action styles in a root level element, it might cause issues in the future as these styles are often mutable.